examples/quick_tutorial.ipynb

```diff
@@ -7,10 +7,12 @@
    "source": [
     "# A quick tutorial\n",
     "\n",
-    "This notebook provides a brief tutorial to modelling vehicle routing problems with PyVRP, introducing some of its most important modelling features.\n",
-    "We will first solve a capacitated VRP, introducing the modelling interface and the most basic components.\n",
-    "We then solve a VRP with time windows, where we introduce the support PyVRP has for problems with time constraints.\n",
-    "Finally, we solve a multi-depot VRP with time windows, to showcase some additional features."
+    "This notebook provides a brief tutorial to modelling vehicle routing problems with PyVRP, introducing some of its most important modelling features:\n",
+    "- We first solve a capacitated VRP, introducing the modelling interface and the most basic components.\n",
```

Comment:
Need a new line here to render itemize

Reply:
Every Markdown parser handles this slightly differently. The one I use locally is fine with this, but clearly whatever Sphinx uses is not. I'll update this to make it work with Sphinx.